packages/toolkit/create/modern.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
       chalk: '@modern-js/utils/chalk',
     },
     dts: false,
+    sideEffects: false,
```

Comment:
Why can't sideEffects be configured in package.json

Reply:
The `sideEffects` configuration here works on third-party dependencies.  When I don't set `sideEffects`, the build product will contain extra code from `@modern-js/utils`. Even if `package.json` of `modern-js/utils` has `sideEffects: false` set, the effect will be same. I think it may be a bug in bundle mode for `@modern-js/module-tools` or `@modern-js/libuild`. I talked to @10Derozan about this bug.

I set `sideEffects: false` in `modern.config.js`, it will be remove unuse code from `@modern-js/utils`.I haven't quite figured out what it is yet, but after trying it this way, if there are no problems, I think it works.

Reply:
get👌🏻